charts/k8s-monitoring/values.yaml

```diff
@@ -345,6 +389,9 @@ grafana-agent:
         mountPath: /etc/grafana-agent-credentials
 
   controller:
+    nodeSelector:
+      kubernetes.io/os: linux
```

Comment:
Last time I tested the nodeSelector on the agent chart it didn't have the desired effect - maybe they fixed that in 0.19.0?

Reply:
Yeah, nodeSelector on the agent Helm chart was added last week:
https://github.com/grafana/agent/commit/d34afce1c808715ed759490c316b7078c1eb5c0c

@davhdavh found this out and I brought the content of their PR into this one.